features/facebookImport/src/model/storage.js

```diff
@@ -37,17 +37,41 @@ export class ZipFile {
     constructor(file, pod) {
         this._pod = pod;
         this._file = file;
+
+        this._entriesSet = null;
+    }
+
+    static async createFor(zipData, pod) {
```

Comment:
IIRC `static` doesn't get compiled by sucrase (i.e. I think it ends up in `dist` as-is), and wouldn't work on iOS 13 if it isn't. The old school alternative would be:

```
export class ZipFile {
    ...
};

ZipFile.createFor = async function (zipData, pod) {
    ...
};
```

Reply:
Also a nit, might wann call it `createWithCache` or `createCached` or something, but that's far less important than the previous comment :D

Reply:
Renamed to createWithCache

Reply:
Also it is defined as `ZipFile. createWithCache  = `